packages/local_auth/local_auth_android/test/local_auth_test.dart

```diff
@@ -79,7 +79,7 @@ void main() {
       ]);
     });
 
-    test('handles emtpy', () async {
```

Comment:
Are we sure we don't need test coverage of "emtpy" handling?

Reply:
I think it's a risk we're going to have to take unfortunately. If we see a flare up of emtpy in the future we can revisit this at that time.